hera_pspec/cli.py

```diff
@@ -0,0 +1,75 @@
+"""A CLI interface for hera_pspec."""
+import typer
+from pathlib import Path
+import warnings
+from rich.console import Console
+import h5py
+import pickle
+import glob
+from tqdm import tqdm
+cns = Console()
+
+app = typer.Typer()
+from . import container
+from .uvpspec import recursive_combine_uvpspec
+
+@app.command()
+def hello():
+    # This is a test command which we need for the CLI interface to be broken into
+    # subcommands (at least two commands need to be defined for it to be used as subc's)
+    cns.print("Hi! :wave:")
+    
+@app.command()
+def fast_merge_baselines(
+    pattern: str = typer.Option(),
+    group: str = typer.Option(),
+    names: list[str] = typer.Option(),
+    outpath: Path = typer.Option(),
+    progress: bool = True,
+    extras: list[str] = None,
+):
+    """Merge a set of hera_pspec files each representing a single baseline, into one.
```

Comment:
Docstring is not extremely clear. Maybe you could detail the arguments or give an example of command line instruction? (personally I wouldn't know what to put under `names`)